__tests__/app.test.js

```diff
@@ -380,5 +380,23 @@ describe('GET /api/articles/:article:id/comments', () => {
              expect (articles).toHaveLength(1)
             expect((articles[0].topic)).toBe('cats')
         })
+       });  
```

Comment:
We need another 200 test for a topic which exists but has no articles. We should receive an empty array.